lib/cc/analyzer/formatters/plain_text_formatter.rb

```diff
@@ -68,6 +85,14 @@ def failed(output)
 
         private
 
+        def report(severity)
+          color = SEVERITY_HUES[severity] || "#333333"
+          str = colorize(" [", "#333333")
+          str += colorize("#{severity}", color)
```

Comment:
Using `<<` instead of `+=` here will be a tiny bit faster, as it will not make a new String object every time.

```
[1] pry(main)> require 'benchmark/ips'
=> true
[2] pry(main)> 
[3] pry(main)> str1 = ""
=> ""
[4] pry(main)> str2 = ""
=> ""
[5] pry(main)> 
[6] pry(main)> Benchmark.ips do |x|
[6] pry(main)*   x.report("+=") { str1 += "a" }  
[6] pry(main)*   x.report("<<") { str2 << "a" }  
[6] pry(main)*   x.compare!  
[6] pry(main)* end  
Calculating -------------------------------------
                  +=     5.054k i/100ms
                  <<   104.102k i/100ms
-------------------------------------------------
                  +=     16.325k (±52.8%) i/s -     70.756k in   5.550502s
                  <<      3.788M (± 5.4%) i/s -     18.947M

Comparison:
                  <<:  3788143.5 i/s
                  +=:    16325.3 i/s - 232.04x slower
```